### PR TITLE
add run name for query constructor

### PR DESCRIPTION
**Pull request by hwchase17**

### CODE REVIEW

File: libs/langchain/langchain/retrievers/self_query/base.py [hunk 1]
```diff
@@ -208,6 +208,7 @@ def from_llm(
         chain_kwargs: Optional[Dict] = None,
         enable_limit: bool = False,
         use_original_query: bool = False,
+        query_constructor_run_name: str = "query_constructor",
```
Comment (baskaryan) [inline libs/langchain/langchain/retrievers/self_query/base.py:211]: imo doesn't need to be an arg, can just hard code

File: libs/langchain/langchain/retrievers/self_query/base.py [hunk 2]
```diff
@@ -208,6 +208,7 @@ def from_llm(
         chain_kwargs: Optional[Dict] = None,
         enable_limit: bool = False,
         use_original_query: bool = False,
```
Comment (eyurtsev) [inline libs/langchain/langchain/retrievers/self_query/base.py:210]: All new named arguments should be introduced with `*` present, to minimize accidental breaking changes
```suggestion
        use_original_query: bool = False,
        *,
```

File: libs/langchain/langchain/retrievers/self_query/base.py (hunk 1 above)
Comment (eyurtsev) [reply libs/langchain/langchain/retrievers/self_query/base.py:211]: If we're using an arg, perhaps we could use a standard arg name argument? (e.g., `run_name`) and it'll be understood that it refers to the top level object

Comment (hwchase17) [reply libs/langchain/langchain/retrievers/self_query/base.py:212]: what if there's multiple different runnables in a constructor?

ill just remove as an arg for now and avoid this question :)

**Pull request merged**